docs/orchestration/kubernetes-yaml/README.md

```diff
@@ -327,7 +345,26 @@ NAME            CLUSTER-IP     EXTERNAL-IP       PORT(S)          AGE
 minio-service   10.55.248.23   104.199.249.165   9000:31852/TCP   1m
 ```
 
-### Step 4: Resource cleanup
+### Update existing Minio StatefulSet
+You can update an existing Minio StatefulSet to use a newer Minio release. To do this, use the `kubectl patch statefulset` command:
+
+```sh
+kubectl patch statefulset minio --type='json' -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/image", "value":"<replace-with-new-minio-image>"}]'
+```
+
+On successful update, you'll see a message like
```

Comment:
Have to rephrase you'll see a message like. Too colloquial. And comes across as you will see something approximately to what we show below.